src/package/store.rs

```diff
@@ -39,7 +41,7 @@ impl PackageStore {
     /// Path to the dependency store
     pub const PROTO_VENDOR_PATH: &'static str = "proto/vendor";
 
-    fn new(root: PathBuf) -> Self {
+    pub(crate) fn new(root: PathBuf) -> Self {
```

Comment:
Why is this needed?

Reply:
Left-over. Reverted in cc82f342bff0e9f93a3168b0cba29001e5f410d1.

Reply:
Actually no. It's needed by the unit test in _buf_yaml.rs_, line 274. I'm going to restore it.